spec/features/admin/remove_user_spec.rb

```diff
@@ -0,0 +1,28 @@
+require "rails_helper"
+
+feature "Remove user" do
+  background do
+    FactoryGirl.create(:ironman)
+  end
+
+  given(:admin) { FactoryGirl.create(:admin) }
+
+  scenario "successful deletion" do
+    visit admin_users_path(as: admin)
+
+    expect(page).to have_content("Tony Stark")
+    expect(page).to have_content("Admin User")
+    find(:xpath, "//tr[contains(.,'Tony Stark')]/td/a", :text => 'Remove').click
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.